girderformindlogger/models/pushNotification.py

```diff
@@ -65,7 +65,7 @@ def createNotification(self, applet, event, creator_id):
 
         users = []
         if 'users' in event['data']:
-            users = [bson.ObjectId(oid=user['_id']) for user in event['data']['users'] if user]
+            users = [bson.ObjectId(oid=user) for user in event['data']['users'] if user]
```

Comment:
it won't work
users array should contain profile id only and this change should be reverted
scheduling won't work for event id
`event_id` should be separated either in data attribute or in `event` 
for request example should be from frontend:
`{
    "schedule": {
        "events": [
            "id": "23h5vhj32v532",  # this is event id we send to frontend for each successfully created event
            data: {
                users: [] # contains profile objects with id for selected users like it was before
                ...
            }
        ]
    }
}`